web-rdfs/source/http_handlers.cc

```diff
@@ -64,46 +56,33 @@ std::string get_path(std::shared_ptr<HttpsServer::Request> request) {
   return path;
 }
 
-std::string get_destination(std::shared_ptr<HttpsServer::Request> request) {
-  std::string destDelim = "&destination=";
-  int idxOfDest = request->query_string.rfind(destDelim) + destDelim.size();
-  std::string dest = request->query_string.substr(idxOfDest);
-
-  LOG(DEBUG) << "Destination given " << dest;
-
-  return dest;
-}
-
-std::string get_user(std::shared_ptr<HttpsServer::Request> request) {
-  std::string userDelim = "&owner=";
-  std::string groupDelim = "&";
-
-  int idxOfUser = request->query_string.rfind(userDelim) + userDelim.size();
-  std::string userAndGroup = request->query_string.substr(idxOfUser);
-
-  int idxOfGroup = userAndGroup.rfind(groupDelim);
-
-  std::string user = userAndGroup.substr(0, idxOfGroup);
-
-  LOG(DEBUG) << "User given " << user;
-
-  return user;
-}
-
-std::string get_group(std::shared_ptr<HttpsServer::Request> request) {
-  std::string groupDelim = "&group=";
-  int idxOfGrp = request->query_string.rfind(groupDelim) + groupDelim.size();
-  std::string group = request->query_string.substr(idxOfGrp);
-
-  LOG(DEBUG) << "Group given " << group;
+std::map<std::string, std::string> parseURL(std::shared_ptr
```

Comment:
`parseQueryString` seems more appropriate

Reply:
I'm pretty sure if a qs param isn't supplied the request will fail because most of the zk methods will return an error if certain parameters aren't given